Project.toml

```diff
@@ -16,16 +16,16 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-Clustering = ">= 0.9.0"
-DataStructures = ">= 0.7.1"
+Clustering = "0.10, 0.11, 0.12, 0.13"
+DataStructures = "0.12, 0.13, 0.14, 0.15, 0.16, 0.17"
 Distances = "0.8"
-ImageFiltering = ">= 0.1.3"
-Images = ">= 0.9"
-LightGraphs = ">= 0.9.0"
-RegionTrees = ">= 0.1.0"
-SimpleWeightedGraphs = ">= 0.0.1"
-StaticArrays = ">= 0.5.0"
-julia = ">= 1.0"
+ImageFiltering = "0.6"
+Images = "0.18, 0.19, 0.20"
+LightGraphs = "1.1, 1.2, 1.3"
```

Comment:
I then realized that "1.1" should be sufficient 😂

Reply:
Sorry, yes, I should have caught that. :+1: